src/components/structures/MatrixChat.tsx

```diff
@@ -501,6 +506,34 @@ export default class MatrixChat extends React.PureComponent<IProps, IState> {
         });
     }
 
+    private onWindowResized = (): void => {
+        // XXX: This is a very unreliable way to detect whether or not the the devtools are open
+        this.warnInConsole();
+    };
+
+    private warnInConsole(): void {
```

Comment:
can this get a leading&trailing throttle on it so it doesn't get spammed during slow(long) resize operations